v3/src/components/data-display/models/data-configuration-model.ts

```diff
@@ -338,7 +338,7 @@ export const DataConfigurationModel = types
 
       selectCasesForCategoryValues(
         primaryAttrRole: AttrRole, primaryValue: string, secondaryValue?: string, extraPrimaryValue?: string,
-        extraSecondaryValue?: string, extend = false
+        extraSecondaryValue?: string, legendCat?: string, extend = false
```

Comment:
Could rename these to `primarySplitValue` and `secondarySplitValue` if you felt motivated but not required. 😉